server/ai_live_video.go

```diff
@@ -83,3 +84,13 @@ func mediamtxSourceTypeToString(s string) (string, error) {
 		return "", errors.New("unknown media source")
 	}
 }
+
+func startControlPublish(control *url.URL, params aiRequestParams) {
+	controlPub, err := trickle.NewTricklePublisher(control.String())
+	if err != nil {
+		slog.Info("error starting control publisher stream=%s err=%v", "err", params.stream, err)
```

Comment:
Should we `return` after this err? Or we can create the pipeline with an empty `ControlPub` here and have a check on [the handler](https://github.com/livepeer/go-livepeer/pull/3255/files#diff-88f3e5816c6c29e92773629b388dfc58ed59a3682ab624c7f3448cfda3106475R487) first and return some 422 or sth

Reply:
I think we should return. Updated.